content/en/quality_gates/_index.md

```diff
@@ -22,35 +25,92 @@ further_reading:
 <div class="alert alert-warning">Quality Gates is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="#" btn_hidden="true" >}}
-Quality Gates is in public beta.
-{{< /callout >}}
-
 ## Overview
 
-Quality Gates allows you to gate your workflows based on signals in Datadog. 
+Quality Gates allow you to control software quality by configuring rules to block substandard code from deployment. You have control over what is merged into the default branch and deployed to production, and can ensure that the code running in production adheres to high quality standards, reducing incidents and minimizing unwanted behaviors.
+
+{{< img src="quality_gates/rule_type.png" alt="A pipeline rule that fails when code coverage for PCT is zero in Quality Gates" style="width:100%" >}}
```

Comment:
I think this screenshot is not a good example. It should fail when the code coverage is "below or equal", not "above or equal"